selfservice/flow/registration/strategy_form_hydrator.go

```diff
@@ -6,8 +6,12 @@ package registration
 import (
 	"encoding/json"
 	"net/http"
+
+	"github.com/pkg/errors"
 )
 
+var ErrBreakRegistrationPopulate = errors.New("skip rest of registration form population")
```

Comment:
Use stdlib errors package or `fmt.Errorf` instead, as otherwise we get the stack trace of the init func here and later don't override it.

Reply:
Also, in general it would be nice to not use errors in cases where everything is fine and we just want to communicate a state up to the caller. But we do this elsewhere already, so I guess fine for now...

Reply:
it's now an stderr (fmt.errorf requires args which we don't have)